cypress/support/cypress-commands/commands.js

```diff
@@ -892,15 +893,7 @@ Cypress.Commands.add('launchApp', (appType, appCallSign, deviceIdentifier) => {
     Cypress.env(CONSTANTS.TEST_TYPE).toLowerCase() == CONSTANTS.MODULE_NAMES.LIFECYCLEAPI ||
     Cypress.env(CONSTANTS.TEST_TYPE).toLowerCase() == CONSTANTS.MODULE_NAMES.LIFECYCLE
   ) {
-    data = {
-      query: {
-        params: {
-          [CONSTANTS.APP_ID]: appId,
-          [CONSTANTS.LIFECYCLE_VALIDATION]: true,
-          [CONSTANTS.APP_TYPE]: appCategory,
-        },
-      },
-    };
+    data.query.params[CONSTANTS.LIFECYCLE_VALIDATION] = true;
```

Comment:
I believe there was one error when we tried to assign these values directly. something happened (check some bug tickets or check with abhishek)

Reply:
yes, if the appType is not certification we will error. Updated the code.